h/tls.h

```diff
@@ -244,6 +279,75 @@ typedef struct CipherMap_tag {
     {0, NULL}\
 };
 
+  typedef struct CurveMap_tag {
+  const char* name;
+  const char* groupId;  //number string
```

Comment:
`number string` is very generic, what does it even mean? Can you be more specific, for example, `I.A.N.A Elliptic curve enumerator`?